### ISSUE TITLE
Creative builders don't work well

### ISSUE THREAD

**Issue by kmecpp** [halvors/Quantum#55]
- [ ] Fusion reactors don't fully form
- [ ] Fission reactors have levers that don't place correctly

There's probably more. The schematics have the potential to make some really cool stuff

**Comment by halvors** [comment]
Is this 1.7.10 or 1.10+? Because schematics have much more potential on 1.10+ with block states.
Also this is mainly meant to be used for debugging, so not a priority really.

**Comment by kmecpp** [comment]
1.7.10

**Comment by halvors** [comment]
Yeah, this is something that should be fixed but have very low priority. Feel free to play around with it :)
